Added meta registration for `aten._foreach_addcdiv_.ScalarList`

awgu opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #123463
* #123362
* #123290
* #122962

This PR fixes https://github.com/pytorch/pytorch/issues/123461 😅

Commit:
Added meta regstiration for `aten._foreach_addcdiv_.ScalarList`

[ghstack-poisoned]

janeyx99 commented on torch/_meta_registrations.py:
```diff
@@ -3337,6 +3337,29 @@ def meta__foreach_addcop__scalar(self, tensor1, tensor2, scalar=1):
+    torch._check(
+        all(isinstance(l, List) for l in [self, tensor1, tensor2, scalars]),
+        lambda: (
+            "All arguments of _foreach_addc*_ must be List[Tensor], "
```
scalars is a List of Scalars though

Commit:
Update on "Added meta registration for `aten._foreach_addcdiv_.ScalarList`"


Trying to fix https://github.com/pytorch/pytorch/issues/123461 😅 

[ghstack-poisoned]
